apps/camera_pipe/camera_pipe.cpp

```diff
@@ -15,13 +17,18 @@ Expr avg(Expr a, Expr b) {
 }
 
 Func hot_pixel_suppression(Func input) {
+    // hot pixel maximum
     Expr a = max(max(input(x-2, y), input(x+2, y)),
                  max(input(x, y-2), input(x, y+2)));
-    Expr b = min(min(input(x-2, y), input(x+2, y)),
-                 min(input(x, y-2), input(x, y+2)));
+    // cold pixel minimum
+    // Expr b = min(min(input(x-2, y), input(x+2, y)),
```

Comment:
It looks to me like the NEON assembly version of this does include cold pixel suppression, while the C++ implementation of FCam does not.

https://github.com/halide/Halide/blob/master/apps/camera_pipe/fcam/Demosaic_ARM.cpp#L260
https://github.com/halide/Halide/blob/master/apps/camera_pipe/fcam/Demosaic.cpp#L133

I think the right fix is probably to add cold pixel suppression to the C++ implementation, rather than remove it from the Halide, otherwise it would need to be removed from the intrinsics version as well.